roles/tpa_single_node/templates/manifests/init/dataset/Deployment.yaml.j2

```
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: apply-dataset-job
+  namespace: "{{ tpa_single_node_namespace }}"
+  labels:
+    app: apply-dataset-job
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: apply-dataset-job
+  template:
+    metadata:
+      labels:
+        app: apply-dataset-job
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: run
+          image: "{{ tpa_single_node_trustification_image }}"
+          imagePullPolicy: IfNotPresent
+          command: [ "/usr/bin/bash" ]
+          args:
+            - "-ce"
+            - |
+              set -e
+              /trust bombastic walker --sink http://sbom-pod --source file:/data/sbom
+              /trust vexination walker --sink http://vexination-api-pod:8080/api/v1/vex --source file:/data/csaf
+          env:
+            - name: OIDC_PROVIDER_CLIENT_ID
+              value: "{{ tpa_single_node_oidc_provider_client_id }}"
+            - name: OIDC_PROVIDER_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  key: client_secret
+                  name: storage_secret
```

```suggestion
            - name: OIDC_PROVIDER_CLIENT_ID
              valueFrom:
                secretKeyRef:
                  key: client_id
                  name: oidc_secret
            - name: OIDC_PROVIDER_CLIENT_SECRET
              valueFrom:
                secretKeyRef:
                  key: client_secret
                  name: oidc_secret               
                  
```

And you need to add the volumemounts and volumes accordingly with "oicd-secret".
Lookup the syntax in bombastic-api deployment manifest if needed.